### PR TITLE
Update BuddyPress Options with 'Group Activity Deletions' info

### PR DESCRIPTION
**Pull request by emaralive**
Added info for the **Group Activity Deletions** options and a few grammatical changes.

bp-documentation issue: https://github.com/buddypress/bp-documentation/issues/299

---
**This Pull Request is for code review only. Please keep all other discussion in the BuddyPress Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the WordPress Core Handbook for more details.**

### CODE REVIEW

File: docs/user/administration/settings/options.md [hunk 1]
```diff
@@ -59,15 +59,20 @@ Enabled by default, this option (if active) will make sure the WordPress user di
...
+Enabled by default, this option leaves the possibility for any group administrator to upload an image and use it as the group's photo. This photo will be used at various places into the community area to better identify your groups singularity.
 
 ### Group cover image upload
 
-Enabled by default, this option leaves the possibility to any group administrator to upload an image and use it as the group's cover image. This image will only be used into the header part of your Groups single pages.
+Enabled by default, this option leaves the possibility for any group administrator to upload an image and use it as the group's cover image. This image will only be used into the header part of your Groups single pages.
+
+### Group Activity Deletions
+
+Enabled by default, this option leaves the possibility for any group administrator or moderstor to delete activity items from their group's activity stream.
```
Comment (imath) [inline docs/user/administration/settings/options.md:74]: Small typo ☺️

```suggestion
Enabled by default, this option leaves the possibility for any group administrator or moderator to delete activity items from their group's activity stream.
```

**Review by imath — CHANGES_REQUESTED** [review]
Hi @emaralive 

Thanks for thèse improvements 👌. I'll SVN commit them shortly.

**Review by imath — APPROVED** [review]
It's ready to be committed!

**Comment by emaralive** [comment]
Ugh, yeah I missed the typo, what's ironic is that earlier I was researching spell checking for GitHub and came across a GitHub Action that might provide that service. Anyway, thanks for making the correction! :smi

**Pull request closed without merging**